feat: add namespace validation in namespace selectors (#2613)
kyma-project/telemetry-manager: docs/user/resources/02-logpipeline.md

| **input.&#x200b;application.&#x200b;namespaces.&#x200b;system**  | boolean | System specifies whether to collect logs from system namespaces. If set to `true`, you collect logs from all namespaces including system namespaces, such as like kube-system, istio-system, and kyma-system. The default is `false`. |
| **input.&#x200b;otlp**  | object | OTLP input configures the push endpoint to receive logs from a OTLP source. |
| **input.&#x200b;otlp.&#x200b;disabled**  | boolean | Disabled specifies if the 'otlp' is deactivated, so that no push-based OTLP signals are collected. The default is `false`. |
| **input.&#x200b;otlp.&#x200b;namespaces**  | object | Namespaces describes whether push-based OTLP signals from specific namespaces are selected. System namespaces are enabled by default. |
| **input.&#x200b;otlp.&#x200b;namespaces.&#x200b;exclude**  | \[\]string | Exclude telemetry data from the specified Namespace names only. By default all namespaces (except system namespaes dependent on input type). An exclude list cannot be specified together with an include list. |

> | **input.&#x200b;otlp.&#x200b;namespaces.&#x200b;include**  | \[\]string | Include telemetry data from the specified Namespace names only. By default all namespaces (except system namespaes dependent on input type). An include list cannot be specified together with an exclude list. |

Reviewer: ```suggestion
| **input.&#x200b;otlp.&#x200b;namespaces.&#x200b;include**  | \[\]string | Include telemetry data from the specified namespace names only. By default, all namespaces (depending on input type: except system namespaces) are included. You can't specify an include list together with an exclude list. |
```
